lib/broccoli/translation-reducer/linter/extract-icu-arguments.js

```diff
@@ -1,4 +1,4 @@
-const parser = require('intl-messageformat-parser');
+const { parse, TYPE } = require('@formatjs/icu-messageformat-parser');
```

Comment:
There seems to have been a syntax change.

https://formatjs.io/docs/icu-messageformat-parser/#usage